tutorials/assets_pipeline/importing_scenes.rst

```diff
@@ -188,327 +188,538 @@ the scene will lead to more consistent results and should be done whenever
+:ref:`class_AnimationPlayer` node.
 
-Additional info:
- * UV2 = The second UV channel for detail textures and baked lightmap textures.
- * Array index = An array of numbers that number each element of the arrays above; i.e. they number the vertices and normals.
+The import options that are visible after changing the import mode to Animation
+Library act the same as when using the Scene import mode. See
+:ref:`doc_importing_3d_scenes_using_the_import_dock` for more information.
 
-In some cases, this might lead to loss of precision, so disabling this option
-may be needed. For instance, if a mesh is very big or there are multiple meshes
-being imported that cover a large area, compressing the import of this mesh(es)
-may lead to gaps in geometry or vertices not being exactly where they should be.
-
-Ensure Tangents
-^^^^^^^^^^^^^^^
-
-If textures with normal mapping are to be used, meshes need to have tangent arrays.
-This option ensures that these are generated if not present in the source scene.
-Godot uses `Mikktspace <http://www.mikktspace.com/>`__ for this,
-but it's always better to have them generated in the exporter.
-
-Storage
-^^^^^^^
-
-Meshes can be stored in separate files (resources) instead of built-in. This does
-not have much practical use unless one wants to build objects with them directly.
-
-This option is provided to help those who prefer working directly with meshes
-instead of scenes.
-
-Light Baking
-^^^^^^^^^^^^
-
-Whether or not the mesh is used in baked lightmaps.
-
-- **Disabled:** The mesh is not used in baked lightmaps.
-- **Enable:** The mesh is used in baked lightmaps.
-- **Gen Lightmaps:** The mesh is used in baked lightmaps, and unwraps a second UV layer for lightmaps.
-
-.. note:: For more information on light baking see :ref:`doc_using_lightmap_gi`.
-
-External Files
-~~~~~~~~~~~~~~
-
-Generated meshes and materials can be optionally stored in a subdirectory with the
-name of the scene.
-
-Animation options
------------------
-
-Godot provides many options regarding how animation data is dealt with. Some exporters
-(such as Blender) can generate many animations in a single file. Others, such as
-3DS Max or Maya, need many animations put into the same timeline or, at worst, put
-each animation in a separate file.
-
-.. image:: img/scene_import3.png
-
-Import of animations is enabled by default.
-
-.. attention::
-
-    To modify animations from an imported 3D scene, you need to change the animation
-    storage option from **Built-In** to **Files** in the Import dock. Otherwise,
-    changes made to animations from Godot will be lost when the project is run.
-
-FPS
-~~~
-
-Most 3D export formats store animation timeline in seconds instead of frames. To ensure
-animations are imported as faithfully as possible, please specify the frames per second
-used to edit them. Failing to do this may result in shaky animations.
-
-Filter Script
-~~~~~~~~~~~~~
+Filter script
```

Comment:
Can someone confirm this feature was removed in 4.0 as part of the import system rewrite? I don't see any references to it in the UI.